pkgs/applications/audio/argotlunar/default.nix

```diff
@@ -0,0 +1,48 @@
+ { pkgs } :
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "argotlunar";
+  version = "2.1.0";
+
+  src = pkgs.fetchgit {
+    url = "https://github.com/mourednik/argotlunar";
+    rev = "1527d5dd753a7af32600ac677337469bbf1d40cd";
+    sha256 = "sha256-U20ZUFrck/Y3GwqEnPZewnUN1YIFucgNsJJTMtWY3wE=";
+  };
+
+  buildInputs = [
+    pkgs.alsa-lib
+    pkgs.freetype
+    pkgs.curl
+    pkgs.pkg-config
+    pkgs.cmake
+    pkgs.xorg.libX11
+    pkgs.xorg.libXrandr
+    pkgs.xorg.libXinerama
+    pkgs.xorg.libxshmfence
+    pkgs.xorg.libXext
+    pkgs.xorg.libXcursor
+  ];
+
+  configurePhase = ''
+    export HOME=$TMPDIR
+    cmake . -B cmake-build
+  '';
+
+  buildPhase = ''
+    cmake --build cmake-build --target all
+  '';
+
+  installPhase = ''
```

Comment:
Does `make install`  not already do the right thing for this software? If it does, you could get rid of this entirely.

If not, you should figure out how to get rid of that `/build/` path because I don't think this is something you can rely on to stay constant. For example I think this might not be a valid path when building without Nix sandboxing enabled.

Reply:
Not sure, for VST and similar plugins you usually need to put it in a system-specific directory. I'll test if I can get rid of this phase as well!

Reply:
I think this is needed (though I did get rid of `/build/`). The original [CMakeLists.txt](https://github.com/mourednik/argotlunar/blob/2.1.0/CMakeLists.txt) file installs the plugin to `$HOME/.vst3`, but from my limited experience with NixOS the expected location for user plugins is `$HOME/.nix-profile/lib/vst3/`.